io_scene_nif/modules/animation/animation_import.py

```diff
@@ -96,7 +96,11 @@ def create_fcurves(action, dtype, dlen, bonename = None):
     if bonename:
         return [action.fcurves.new(data_path = 'pose.bones["'+bonename+'"].'+dtype, index = i, action_group = bonename) for i in range(dlen)]
     else:
-        return [action.fcurves.new(data_path = dtype, index = i, action_group = "LocRotScale") for i in range(dlen)]
+        if any(s in dtype for s in ("rotation","location","scale")):
```

Comment:
Any is intended here, as the fcurve should be lumped into "LocRotScale" if it is any of the given types. It can never be all at once. This is so that not transformaing animations can also use this function, such as visibility or material anims.

Reply:
Documented it in the code ;)